README.md

```diff
@@ -1,3 +1,68 @@
-# method-scope
+# Method Scope
 
-local vs. method scope
+## Objectives
+1. Understand the concept of scope. 
+2. Learn the scope of a method. 
+3. Learn the scope of a local variable. 
+
+## What is Scope?
+
+Methods in ruby create their own scope. "Scope" refers to the areas of your program in which certain data is available to you. **Any local variable created outside of a method will be unavailable inside of a method. In addition, local variables created inside of a method 'fall out of scope' once you're outside the method.**
```

Comment:
Quick intro paragraph (feel free to re-word):

Naming variables is hard. In general we want our variable names to be descriptive of what they hold and still concise. To make this a bit easier, all programming languages have what is called scope. Scope means that not all variable exist everywhere. IF they did then some variable that you'd be writing in file A, would accidentally overwrite a variable made by your friend in file B. Having a scope for variables allows you to have greater confidence when creating variables that you aren't overwriting someoene elses work.